Assets/Scripts/AsteroidCollisionHealthEffect.cs

```diff
@@ -1,5 +1,4 @@
-﻿using System;
-using UnityEngine;
+﻿using UnityEngine;
 
 public class AsteroidCollisionHealthEffect : HealthEffectBehavior
 {
```

Comment:
I know this wasn't touched by this PR, but if you could quickly fix the typo, that'd be great:
On line 11 in your version, size is set to `size.magnitude` when it really should be `size.x`. The calculation is a bit "off" and skews towards higher than expected.

Reply:
Okay I can change it for now but honestly that calculation should be completely different anyway. The damage dealt on collision should be proportional to the speed of impact, not the size of the asteroid.